src/modules/quick_method_module.f90

```diff
@@ -893,6 +894,9 @@ subroutine check_quick_method(self,io,ierr)
                 self%OPT = .false.
             endif
 
+            ! tighten XCCutoff if diffuse functions exist
+            if(self%diffuse_basis_funcs .and. self%DFT) self%XCCutoff=self%XCCutoff*0.1d0
```

Comment:
I think this should only be done for default values. I.e. if a user requests XCCutoff in the input, then it should not be changed.